spec/status.yml

```
@@ -159,7 +159,7 @@ components:
       get:
         tags:
         - 'status'
-        operationId: "authenticatorStatus"
+        operationId: "authenticator"
```

Since only one authenticator uses this route right now, maybe this should just be hardcoded to:
```suggestion
        operationId: "gcpAuthenticator"
```

Then you can also remove the `{authenticator}` in the path in favor of hardcoding `authn-gcp`

---

this suggestion stands, it's not just `authenticator` it's specific to GCP